finality-aleph/src/network/service.rs

```diff
@@ -641,8 +605,8 @@ mod tests {
 
         test_data
             .mock_io
-            .messages_for_user
-            .unbounded_send((message.clone().into(), DataCommand::SendTo(peer_id.clone())))
+            .data_for_network
```

Comment:
Now this test should probably be named `test_sends_validator_data`

Reply:
Ha, that definitely would have been confusing, changed.